libcxx/test/std/utilities/meta/meta.unary/dependent_return_type.pass.cpp

```diff
@@ -168,3 +168,6 @@ void instantiate() {
   void_t<int>();
 #endif
 }
+
+// This is not a .compile.pass.cpp, because GCC only complains in CodeGen about incorrect builtins usage
```

Comment:
```suggestion
// This is not a .compile.pass.cpp because we want to ensure that GCC doesn't complain about incorrect builtins usage, which only happens during CodeGen.
```